locales/metabase.pot

```diff
@@ -0,0 +1,138 @@
+# #-#-#-#-#  metabase-backend.pot (metabase)  #-#-#-#-#
```

Comment:
We'll need this initially, but I think we'll want to have something automated in place to generate these files and check them in. If not, pretty much every PR will conflict on this file. Any addition or removal of a line above/below a string will cause the line number to change an thus a change to this file.

Reply:
So the idea is every time `master` is updated this automated thing would run `./bin/i18n/update-translations` and commit the changes?

Reply:
how would this script get ran, as a Git commit hook?

Reply:
@tlrobinson Is that script where the .po* files are being generated? Taking a look at the script in the PR that's what it looks like it's doing. The important point there is not relying on humans to remember to run that script and to get it out of PRs so that it doesn't cause merge conflicts.

@camsaul Right, I think we want a push hook that just listens on master described [here](https://github.com/metabase/metabase/issues/5893)

Reply:
That would totally bust development for people on Windows (including myself once or twice a week and several contributors) 😞

Reply:
Not sure I understand, what would bust windows developers?

Reply:
running bash scripts as part of git hooks

Reply:
I see where the misunderstanding is. It's a hook on the central repo, not a local git repo hook, see my link above. When a commit is pushed (and on master) we would have a process running on a separate server that would get POSTed to. That would cause the script to run on that server, check in the changes. 

The intent is to keep these kinds of things away from developers day to day operations. Since developers aren't messing with it, we won't have merge conflicts (or windows compatiblity issues).

Reply:
oh ok. Makes sense 👍